internal/evaluator/conftest_evaluator.go

```diff
@@ -120,8 +122,10 @@ func (c *CheckResults) trim() {
 	}
 }
 
+type Data map[string]any
```

Comment:
Does it make sense for this to be `map[string]json.RawMessage` ?

https://pkg.go.dev/encoding/json#RawMessage

Reply:
What we get out of Conftest is something like
`map[string]map[string]map[string]int64`, so that would not fit...